### DISCUSSION TITLE
Feature Request: Add app to ignore/exclusion list

### DISCUSSION THREAD

**Discussion by jin-ds** [will-stone/browserosaurus#592] [Ideas]
Hi there!

Love the app!

I would like to exclude some apps from showing the popup and just use the default browser, thanks!

**Comment by kansasmarten** [comment]
I feel like this used to be an option, but I can't remember for sure. Would love to see some toggles!

**Comment by will-stone** [comment]
Thanks for using Browserosaurus! As @kansasmarten mentioned, yes it did used to be an option, but no configuration is always the best route for app maintenance. Move your less used browsers/apps to the bottom of the list in the Preferences, then shrink your picker window height, et voilà!

**Reply by ponugoti** [reply]
@will-stone do you mind sharing if there is a straightforward way to add this functionality on our own? Or maybe a link to the state of the repo when this _was_ as feature?

PS thanks for the awesome app :)

**Reply by will-stone** [reply]
It was a long time ago, so I wouldn’t use that code if I were you. If having apps available but not visible below the scroll really offends you that much, I would suggest building the app locally, that way you are free to remove (or add) any apps you’d like from the supported apps list.
